vmm/src/api/openapi/cloud-hypervisor.yaml

```diff
@@ -126,7 +126,7 @@ paths:
 components:
   schemas:
 
-    VmmInfo:
+    VmmPingResponse:
```

Comment:
@jcvenegas 
You seems to forget to replace the reference to `VmmInfo` with `VmmPingResponse` along with the change from `/vmm.info` to `/vmm.ping` : `$ref: '#/components/schemas/VmmInfo'` ->`$ref: '#/components/schemas/VmmPingResponse'` .